### PR TITLE
Create separate task for each contact

### PR DESCRIPTION
**Pull request by adammeyer**
Helpscout issue 1356

### CODE REVIEW

**Comment by adammeyer** [comment]
@soberstadt Can you review

File: app/assets/javascripts/angular/directives/appealsList.js [hunk 1]
```diff
@@ -83,10 +83,11 @@ angular.module('mpdxApp')
                             };
 
                             $scope.createTask = function(taskType, inputContactsObject){
+                                $scope.creatingBulkTasks = true;
                                 var contactsObject = [];
                                 angular.forEach(inputContactsObject, function(value, key) {
```
Comment (soberstadt) [inline app/assets/javascripts/angular/directives/appealsList.js]: What about using something like _.keys (https://lodash.com/docs#keys) here?

Maybe `contactsObject = _.keys(inputContactsObject)`? use _.compact as needed.

**Comment by soberstadt** [comment]
Interesting use of recursion. Other than that one little thing, :+1:

**Comment by adammeyer** [comment]
@soberstadt how's this?

**Pull request merged**